features/htcondor/client/fix-blah.pan

```diff
@@ -0,0 +1,23 @@
+unique template features/htcondor/client/fix-blah;
+
+variable HTCONDOR_FIX_JOB_CANCEL?=true;
+
+# Fixme: a bug in the blah RPM
+include {'components/filecopy/config'};
```

Comment:
Can you use the new pan format by removing the braces?

Reply:
Hi Jerome,
I was waiting for the system check to run before fixing all the typos and formatting problems. 
This one is fixed in the new version.